junit-test/shared/src/test/scala/scala/scalanative/junit/IgnoreAllTest.scala

```diff
@@ -0,0 +1,14 @@
+package scala.scalanative.junit
+
+import org.junit._
+
+import scala.scalanative.junit.utils.JUnitTest
+
+@Ignore
+class IgnoreAllTest {
```

Comment:
Consider adding a `throw new Error("unreachable")` in the body of the class, to ensure that the test class is never instantiated if it has the `@Ignore` annotation.